sdk/documenttranslation/azure-ai-documenttranslation/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Release History
```

Comment:
does the folder need to match the namespace? just wondering if this should be in `translation`  or something else.
For now it is fine

Reply:
Still part of the larger namespace issue. Might move around later.